packages/core/entity/FileDetail/RenderZarrThumbnailURL.ts

```diff
@@ -0,0 +1,101 @@
+import * as zarr from "@zarrita/core";
+import { FetchStore } from "@zarrita/storage";
```

Comment:
Unused?

Reply:
I think all of these are used. Am I missing something? zarr is used for accessing the root. FetchStore is used for accessing the store from url and zarrGet is used to create a lowest resolution view.